pkg/gofr/gofr_test.go

```diff
@@ -504,3 +504,107 @@ func Test_AddCronJob_Success(t *testing.T) {
 
 	assert.Truef(t, pass, "unable to add cron job to cron table")
 }
+
+func TestStaticHandler(t *testing.T) {
+	// Generate a public directory endpoint
+	directory := "./public"
```

Comment:
You could use the constant I talked about

Reply:
used the constat